routes/index.tsx

```diff
@@ -431,7 +432,12 @@ function InstallSection({ url }: { url: URL }) {
         </a>{" "}
         (Mac):
       </p>
-      <CodeBlock language="bash" code="brew install deno" url={url} />
+      <CodeBlock
+        language="bash"
+        code="brew install deno"
+        url={url}
+        enableCopyButton={true}
```

Comment:
nitpick
```suggestion
        enableCopyButton
```

Reply:
Ah I didn't even know this was possible, thanks!